server/http.go

```diff
@@ -348,10 +349,37 @@ func (p *Plugin) postMeeting(creator *model.User, meetingID int, channelID strin
 	if appErr = p.storeMeetingPostID(meetingID, createdPost.Id); appErr != nil {
 		p.API.LogDebug("failed to store post id", "err", appErr)
 	}
-
 	return nil
 }
 
+func (p *Plugin) askUserPMIMeeting(userID string, channelID string) {
```

Comment:
give a better name for this according to its functionality

Reply:
I think it is most appropriate name